sandbox/gas/tube/go

```diff
@@ -64,17 +65,46 @@ class DiaphragmIAnchor(sc.MeshAnchor):
         svr.soln[:,2].fill(0.0)
         if svr.ndim == 3:
             svr.soln[:,3].fill(0.0)
-        svr.soln[:,svr.ndim+1].fill(self.p1/(gamma-1))
+        # pressure
+        svr.soln[:,svr.ndim+1].fill(self.p1)
         # set.
-        slct_int = svr.blk.clcnd[:,2] > 0.5
-        slct_gst = svr.blk.gstclcnd[:,2] > 0.5
+        tube_middle = self.tube_height / 2
+        slct_int = svr.blk.clcnd[:,2] > tube_middle
+        slct_gst = svr.blk.gstclcnd[:,2] > tube_middle
         slct = np.concatenate((slct_gst, slct_int))
         svr.soln[slct,0] = self.rho2
-        svr.soln[slct,svr.ndim+1] = self.p2/(gamma-1)
+        svr.soln[slct,svr.ndim+1] = self.p2
```

Comment:
This commit https://github.com/solvcon/solvcon/pull/182/commits/baa6c306617deac09551aeac49a1352e133646f1 fixed the initialized data with total energy instead of pressure.